packages/create-discord-bot/package.json

```diff
@@ -0,0 +1,71 @@
+{
+	"name": "create-discord-bot",
+	"version": "0.1.0",
+	"description": "A simple way to create a startup Discord bot.",
+	"keywords": [
+		"discord",
+		"discord-api",
+		"discord.js",
+		"create-bot",
+		"init-bot",
+		"init-discord",
+		"boilerplate"
+	],
+	"homepage": "https://discord.js.org",
+	"bugs": {
+		"url": "https://github.com/discordjs/discord.js/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/discordjs/discord.js.git"
+	},
+	"license": "Apache-2.0",
+	"contributors": [
+		"Crawl <icrawltogo@gmail.com>",
+		"SpaceEEC <spaceeec@yahoo.com>",
+		"Vlad Frangu <kingdgrizzle@gmail.com>",
+		"Aura Román <kyradiscord@gmail.com>",
+		"Jiralite <jiralite@live.co.uk>"
+	],
+	"bin": "./dist/create-discord-bot.mjs",
+	"directories": {
+		"lib": "src"
+	},
+	"files": [
+		"dist",
+		"template"
+	],
+	"scripts": {
+		"build": "tsup",
+		"changelog": "git cliff --prepend ./CHANGELOG.md -u -c ./cliff.toml -r ../../ --include-path 'packages/create-discord-bot/*'",
+		"format": "prettier --write . && cross-env TIMING=1 eslint src --ext .mjs,.js,.ts --fix --format=pretty",
+		"lint": "prettier --check . && cross-env TIMING=1 eslint src --ext .mjs,.js,.ts --format=pretty",
+		"prepack": "yarn build && yarn lint"
```

Comment:
Missing a release script with cliff-jumper, also the ordering is wrong compared to other package.json files.